### PR TITLE
[4.0] Empty catch statement

### PR DESCRIPTION
**Pull request by brianteeman**
My ide is telling me that it is bad to have an empty catch statement and if there is one then it should have a comment. Following the example of @roland-d  @mbabker and others I added a comment

### CODE REVIEW

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 1b257305c44f825e444af59c83de654c1f808c45<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27638">issues.joomla.org/tracker/joomla-cms/27638</a>.</sub>

File: libraries/src/Language/LanguageHelper.php [hunk 1]
```diff
@@ -653,6 +653,7 @@ public static function parseLanguageFiles($dir = null)
 				}
 				catch (\RuntimeException $e)
 				{
+					// Ignore it
```
Comment (mbabker) [inline libraries/src/Language/LanguageHelper.php:656]: I generally try to use a comment to explain why the catch block is basically no-opped, that way someone coming along later knows why it’s empty and doesn’t assume it’s an error there’s no code there.

Comment (brianteeman) [reply libraries/src/Language/LanguageHelper.php:656]: Search the code. I just copied exactly what you have done :)

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 1b257305c44f825e444af59c83de654c1f808c45<br /><br />Code review.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27638">issues.joomla.org/tracker/joomla-cms/27638</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27638">issues.joomla.org/tracker/joomla-cms/27638</a>.</sub>

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**